src/cunumeric/execution_policy/execution_policy_helpers.h

```diff
@@ -0,0 +1,11 @@
+#ifndef SRC_CUNUMERIC_EXECUTION_POLICY_EXECUTION_POLICY_HELPERS_H_
```

Comment:
Use `#pragma once` here as well

Reply:
Well this file is no longer necessary. @jjwilke please remove this file from the PR.

Reply:
Done.